stdlib/public/Darwin/CoreGraphics/CoreGraphics.swift

```diff
@@ -49,7 +49,7 @@ extension CGColor {
 }
 
 public protocol _CGColorInitTrampoline {
-#if os(macOS)
```

Comment:
What’s up with these overlay changes?

Reply:
Good catch, had to do it to compile on my machine--will remove them.

Reply:
And thanks for looking at it so quickly!